examples/dlp_api_example/main.tf

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  region = "${var.region}"
+}
+
+locals {
+  gcs_bucket_name = "tmp-dir-bucket-${random_id.random_suffix.hex}"
```

Comment:
This example won't compile since the resource "random_id" isn't created anywhere.
Add this
`resource "random_id" "random_suffix" {
  byte_length = 4
}`

and test again.